katalog/cerebro/README.md

```diff
@@ -5,12 +5,12 @@ common tasks on ElasticSearch cluster via Cerebro's web interface.
 
 ## Requirements
 
-- Kubernetes >= `1.14.0`
+- Kubernetes >= `1.16.0`
```

Comment:
Silly question: why do we need kubernetes >= 1.16.0 for Cerebro?

Reply:
It is not a strict requirement. As we are testing all the packages from 1.16 up to 1.19, we can not ensure it will work in version < 1.16. This is why I set Kubernetes 1.16 as a requirement